openrtb_ext/imp_taboola.go

```diff
@@ -7,4 +7,6 @@ type ImpExtTaboola struct {
 	TagId           string   `json:"tagid"`
 	BCat            []string `json:"bcat"`
 	BAdv            []string `json:"badv"`
+	PageType        string   `json:"pageType"`
+	Position        *int8    `json:"position"`
```

Comment:
Is there any particular reason for using 8 bit integer here instead of using an int? If we use 8 bit integer and if the value of position exceeds 8 bits (i.e 127), integer overflow will happen and that would lead to unexpected results.

Reply:
Thanks for your comment, pushed a fix